### PR TITLE
fix exponential notation

### LINKED ISSUE #1047: double.ToString format specification wrong
The table describing the resulting format contains an error. The symbol for exponential notation is given as lower-case 'e'. However, the actual symbol that is produced is capital 'E'. All examples on the page also use 'E'. This is the case for both the parameterless overload and the one taking IFormatProvider.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 41a55742-f95a-f0d2-8b5d-7d47f6d0a0c6
* Version Independent ID: ce5f86d2-57f1-0fcc-caf2-bdc527d20a1c
* Content: [Double.ToString Method (System)](https://docs.microsoft.com/en-us/dotnet/api/system.double.tostring?redirectedfrom=MSDN&view=netframework-4.7.2#System_Double_ToString)
* Content Source: [xml/System/Double.xml](https://github.com/dotnet/dotnet-api-docs/blob/master/xml/System/Double.xml)
* Product: **dotnet-api**
* GitHub Login: @dotnet-bot
* Microsoft Alias: **dotnetcontent**

### PR DESCRIPTION
**Pull request by mairaw**
Fixes #1047

@carlossanlop @tarekgh do you know who could validate these changes?

### CODE REVIEW

**Comment by carlossanlop** [comment]
@tannergooding @tarekgh can you please help verify these changes?

**Comment by tarekgh** [comment]
CC @tannergooding

File: xml/System/Double.xml [hunk 1]
```diff
@@ -3232,7 +3232,7 @@ If a separator is encountered in the `s` parameter during a parse operation, and
   
  The return value can be <xref:System.Globalization.NumberFormatInfo.PositiveInfinitySymbol%2A>, <xref:System.Globalization.NumberFormatInfo.NegativeInfinitySymbol%2A>, <xref:System.Globalization.NumberFormatInfo.NaNSymbol%2A>, or a string of the form:  
   
- [sign]integral-digits[.[fractional-digits]][e[sign]exponential-digits]  
+ [sign]integral-digits[.[fractional-digits]][E[sign]exponential-digits]
```
Comment (tannergooding) [inline xml/System/Double.xml]: This is specifically for the `ToString()` overload that takes no parameters, correct?

Comment (mairaw) [reply xml/System/Double.xml]: This explanation is present in two overloads:
image

Comment (tannergooding) [reply xml/System/Double.xml]: Then LGTM.

I just wanted to ensure this wasn't being done on any method that took a `string format`, as that will use `E` for an uppercase format specifier (such as `G`, `R`, `F`, `N`, or `E`) and `e` for a lowercase format specifier (such as `g`, `r`, `f`, `n`, or `e`).

**Pull request merged**